pkg/engine/validate/validate.go

```diff
@@ -11,14 +11,19 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/nirmata/kyverno/pkg/engine/anchor"
 	"github.com/nirmata/kyverno/pkg/engine/operator"
+	"github.com/nirmata/kyverno/pkg/engine/common"
 )
 
 // ValidateResourceWithPattern is a start of element-by-element validation process
 // It assumes that validation is started from root, so "/" is passed
 func ValidateResourceWithPattern(log logr.Logger, resource, pattern interface{}) (string, error) {
-	path, err := validateResourceElement(log, resource, pattern, pattern, "/")
+	// newAnchorMap - to check anchor key has values 
+	ac := common.NewAnchorMap()
+	path, err := validateResourceElement(log, resource, pattern, pattern, "/", ac)
 	if err != nil {
-		return path, err
+		if !ac.IsAnchorError() {
+			return path, err
+		}
```

Comment:
What's the `else` here? How are we handling it?

Reply:
if there is any errors in the anchors then it will succeed the validation and skip further validations of any other recursive function.